### PR TITLE
feat: server support force reload page

### PR DESCRIPTION
**Pull request by DiamondYuan**
##### Checklist

- [ ] `npm test` passes
- [ ] tests are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/umi/ISSUE_URL

### CODE REVIEW

**Comment by DiamondYuan** [comment]
@sorrycc bundler-webpack/bundled 里面的文件是怎么编译的。

File: packages/bundler-webpack/src/webpackHotDevClient/webpackHotDevClient.ts [hunk 1]
```diff
@@ -20,7 +20,11 @@ function handleHashChange(hash: string) {
   mostRecentCompilationHash = hash;
 }
 
-function handleSuccess() {
+function handleSuccess(data?: { reload: boolean }) {
+  if (data && data.reload) {
+    window.location.reload();
+    return
```
Comment (sorrycc) [inline packages/bundler-webpack/src/webpackHotDevClient/webpackHotDevClient.ts:26]:
```suggestion
    return;
```

**Comment by sorrycc** [comment]
> @sorrycc bundler-webpack/bundled 里面的文件是怎么编译的。

应该是在 bundler-webpack 下执行 `./bin/bundler-webpack build`。

**Comment by ycjcl868** [comment]
这个需要单独编译一次，

**Pull request merged**